src/test/kotlin/io/github/anki/anki/controller/DecksControllerTest.kt

```diff
@@ -120,6 +118,15 @@ class DecksControllerTest @Autowired constructor(
                 .assertNext {
                     it.toDeck().toDto() shouldBe createdDeck
                 }
+                .verifyComplete()
+            StepVerifier
```

Comment:
As I understand you can do deckRepository.findById(createdDeck.id.toObjectId()) and then in assertNext call two asserts it.toDeck().toDto() shouldBe createdDeck and it.event shouldBe DeckEventType.CREATED.name

Reply:
it is two different repositories (with decks and with deck events) so i have to make two StepVerifiers